common/src/main/java/com/tencent/rss/common/config/ConfigOptions.java

```diff
@@ -54,6 +57,8 @@
  * }</pre>
  */
 public class ConfigOptions {
+  private static final String LIST_SPILTTER = ",";
```

Comment:
Do we move to class ListConfigOptionBuilder?

Reply:
Done